disabled-packages/ldc/build.sh

```diff
@@ -34,7 +34,7 @@ termux_step_post_extract_package () {
 		$TERMUX_PKG_CACHEDIR/dmd.${DMD_COMPILER_VERSION}.linux.tar.xz \
 		e48783bd91d77bfdcd702bd268c5ac5d322975dd4b3ad68831babd74509d2ce9
 
-	sed "s#\@TERMUX_C_COMPILER\@#$TERMUX_STANDALONE_TOOLCHAIN/bin/$TERMUX_HOST_PLATFORM-clang#" \
+	sed "s#\@TERMUX_C_COMPILER\@#$TERMUX_TOPDIR/_lib/$TERMUX_NDK_VERSION-$TERMUX_ARCH-$TERMUX_PKG_API_LEVEL-v8/bin/$TERMUX_HOST_PLATFORM-clang#" \
```

Comment:
The definition of `TERMUX_STANDALONE_TOOLCHAIN` was moved ahead of this step, so I can't use it in the host build anymore.  Can you move it back, so host builds can use the cross-compilation toolchain if needed?  If not, I'll have to define it here myself, which will be tedious to keep updated.